miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppScheme.kt

```diff
@@ -48,4 +47,15 @@ internal class MiniAppScheme private constructor(miniAppId: String) {
         it.data = url.toUri()
         context.startActivity(it)
     }
+
+    internal fun openMaiComposer(context: Context, url: String) = Intent(Intent.ACTION_SEND).let {
+        val mail = MailTo.parse(url)
```

Comment:
Can this throw an error if the string isn't a proper mail URI? Maybe it should be inside try/catch?

Reply:
We only call this function when it's `mailto:` uri. In addition, when there is any invalid argument, the map header of `mail` will have null value which is allowed in android mail intent.